README.md

```diff
@@ -263,10 +267,28 @@ default. See https://www.lightbend.com/blog/scala-inliner-optimizer for more det
 
 See [tests/README.md](tests/README.md).
 
+### Using Trogdor for testing ###
+
+We use Trogdor as a test framework for Apache Kafka. You can use it to run benchmarks and other workloads.
+
+See [trogdor/README.md](trogdor/README.md).
+
 ### Running in Vagrant ###
 
 See [vagrant/README.md](vagrant/README.md).
 
+### Release Kafka ###
```

Comment:
This is not something a regular person can do and hence it doesn't seem like something we should link from the main readme.

Reply:
The original idea was to list all README.md files in the root directory, without filtering the content for specific readers. I'm fine to remove the "Release Kafka", and the "Official Documentation" could be removed too as the file "docs/README.md" has nothing to read